tests/bluetooth/bsim_bt/bsim_test_per_adv/src/per_adv_advertiser.c

```diff
@@ -276,6 +313,31 @@ static void main_per_adv_conn_privacy_advertiser(void)
 	PASS("Periodic advertiser passed\n");
 }
 
+static void main_per_adv_long_data_advertiser(void)
+{
+#if (CONFIG_BT_PER_ADV_SYNC_BUF_SIZE > 0)
```

Comment:
`CONFIG_BT_PER_ADV_SYNC_BUF_SIZE` only applies to the sync receiver, right? What's the argument for guarding the advertiser side?

Reply:
Could use either of these for advertiser:
`CONFIG_BT_CTLR_ADV_DATA_LEN_MAX`
`CONFIG_BT_CTLR_ADV_DATA_CHAIN`
which I did initially use the ADV_DATA_LEN_MAX > 31 but set them both to the SYNC_BUF_SIZE, since it was only used in this test and they both used the same conf file. But for readability I can change it back